python/python/glide/async_commands/core.py

```diff
@@ -1232,3 +1234,104 @@ async def zrem(
             int,
             await self._execute_command(RequestType.Zrem, [key] + members),
         )
+
+    @dataclass
+    class PubSubMsg:
```

Comment:
docstring example for class' args in python:
```
@dataclass
class TestClass:
    """This is a test class for dataclasses.

    This is the body of the docstring description.

    Args:
        var_int (int): An integer.
        var_str (str): A string.

    """

Reply:
what style it it? I find it more complicated then the proposed form. What is the difference?

Reply:
this is one of python's most common docstring formats that we use throughout the project
https://www.sphinx-doc.org/en/master/usage/extensions/example_google.html#example-google

Reply:
there is no example for the dataclass in the sphinx doc, "args" are also not applicable to dataclass. I`ll follow the chat-gtp proposed style

Reply:
done